README.md

```diff
@@ -79,4 +79,12 @@ Host bootstrap
     UserKnownHostsFile /dev/null
 ```
 
+## ENV
+
+Following environment variables can be used to change default value of bundle generation.
```

Comment:
"The following environment variables can be used to change the default values of bundle generation"